cmd/pipeline/bootstrap.yaml

```diff
@@ -63,10 +63,21 @@ edge-reporter:
   target:
     url: "${OPENAPI_PUBLIC_URL}"
     auth_token: "${OPENAPI_TOKEN}"
-
+erda.apps.gallery: { }
```

Comment:
reverse depends it not permitted.

Reply:
adjusted, gallery listen extension event